sdk/communication/AzureCommunication/Source/Generated/MicrosoftTeamsUserIdentifierModel.swift

```diff
@@ -0,0 +1,67 @@
+// --------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+// --------------------------------------------------------------------------
+
+import AzureCore
+import Foundation
+// swiftlint:disable superfluous_disable_command
+// swiftlint:disable identifier_name
+// swiftlint:disable line_length
+// swiftlint:disable cyclomatic_complexity
+
+/// A Microsoft Teams user.
+public struct MicrosoftTeamsUserIdentifierModel: Codable, Equatable {
+    // MARK: Properties
+
+    /// The Id of the Microsoft Teams user. If not anonymous, this is the AAD object Id of the user.
+    public let userId: String
+    /// True if the Microsoft Teams user is anonymous. By default false if missing.
+    public let isAnonymous: Bool?
+    /// The cloud that the Microsoft Teams user belongs to. By default 'public' if missing.
+    public let cloud: CommunicationCloudEnvironmentModel?
+
+    // MARK: Initializers
+
+    /// Initialize a `MicrosoftTeamsUserIdentifierModel` structure.
+    /// - Parameters:
+    ///   - userId: The Id of the Microsoft Teams user. If not anonymous, this is the AAD object Id of the user.
+    ///   - isAnonymous: True if the Microsoft Teams user is anonymous. By default false if missing.
+    ///   - cloud: The cloud that the Microsoft Teams user belongs to. By default 'public' if missing.
+    public init(
+        userId: String, isAnonymous: Bool? = false, cloud: CommunicationCloudEnvironmentModel? = .public
```

Comment:
The original init generated here was actually 
```
public init( userId: String, isAnonymous: Bool? = nil, cloud: CommunicationCloudEnvironmentModel? = .nil)
```
Which actually isn't correct according to the documentation so I changed it up manually. But this may occur again the next time we generate it. Not sure what to do about this issue. @sacheu @tjprescott

Reply:
Is `.nil` an option?  You should file an issue in autorest.swift repo.

Reply:
Sorry I think this is my mistake it would have just been `nil` I don't think `.nil` would have worked.